pkg/systemd/quadlet/quadlet.go

```diff
@@ -628,6 +630,11 @@ func ConvertContainer(container *parser.UnitFile, names map[string]string, isUse
 		podman.addf("--shm-size=%s", shmSize)
 	}
 
+	entrypoint, hasEntrypoint := container.Lookup(ContainerGroup, KeyEntrypoint)
+	if hasEntrypoint {
```

Comment:
Please also check that the value is not an empty string
```suggestion
	if hasEntrypoint && len(entrypoint) > 0 {
```

Reply:
Entrypoint is allowed to be empty, in fact it may be required to use `--entrypoint=` to unset an entrypoint from the container image.

Reply:
Oh, didn't think about it. OK, resolving my comment